shared/routes/user.routes.ts

```diff
@@ -48,7 +45,8 @@ export const zUserRecruteurRoutes = {
       },
       securityScheme: {
         auth: "cookie-session",
-        role: "all",
+        access: "admin",
+        ressources: {},
```

Comment:
Attention OPCO dashboard

Les opcos get les users

Reply:
Retour sur les autorisations : 

Pour moi opco credential (23) et opco sont redondants : 

I do not have user:manage permission (1)
           ✓ on user CFA from my Opco
         ✓ I do not have admin permission (1)
           ✓ on user CFA from my Opco
—> retirer la partie « from my Opco » 

@moroine 
@remy-auricoste certaines parties pourront être adaptés lors de la livraison du multicompte

Reply:
Même remarque pour la partie opco —> retirer « from my Opco » comme évoqué à l’oral